### ISSUE TITLE
There are no *.exe files.

### ISSUE THREAD

**Issue by evil-pechenka** [jaydenmilne/steamsync#32]
![2023-05-12_19-04-33-%t](https://github.com/jaydenmilne/steamsync/assets/109512933/64e7c38c-ccd3-49a9-a56f-d42029c956a2)
Maybe I don't understand something, but there are no exe files in the releases. :(

It's very sad, it's a great program.

**Comment by jaydenmilne** [comment]
Are you trying to use the desktop version? I haven’t updated that in a while.

Knowing people use it though might be enough motivation to update it though, tbd

**Comment by evil-pechenka** [comment]
Yes, I was using version 0.3, but it stopped working with Epic Games, so I tried to download the new version. However, there are no more exe files. And since I'm not a programmer, for me, it means there is nothing at all.

**Comment by xRoseQuartzx** [comment]
I'm in the same boat as Evil-Pechenka.  Though I attempted to learn enough Python to try and build the gui from whats in the source, but didn't have much luck.

**Comment by jaydenmilne** [comment]
Apologies for this, I threw together the gui quick and pretty much immediately forgot about it / how to fix it
